cypress/e2e/contracts/multisig.spec.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 @paritytech/contracts-ui authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import {
+  beforeAllContracts,
+  assertUpload,
+  assertMoveToStep2,
+  assertMoveToStep3,
+  assertContractRedirect,
+  assertInstantiate,
+} from '../../support/util';
+export function assertFormField(argument: string, type: string, inputType = 'text') {
+  cy.get(`.form-field.${argument}`)
+    .scrollIntoView()
+    .within(() => {
+      cy.contains(`${argument}: ${type}`).should('be.visible');
+      cy.get(`input[type='${inputType}']`).should('be.visible').and('be.empty');
+    });
+}
+describe('Multisig Contract', () => {
+  before(() => {
+    beforeAllContracts();
+  });
+
+  it('contract file uploads', () => {
+    assertUpload('multisig.contract');
+  });
+
+  it('moves to step 2', () => {
+    assertMoveToStep2();
+  });
+
+  it('displays `requirement: u32` input correctly', () => {
+    // assertFormField('requirement', 'u32', 'number');
+    cy.get('.form-field.requirement')
+      .scrollIntoView()
+      .within(() => {
+        cy.contains('requirement: u32').should('be.visible');
+        cy.get("input[type='number']").should('be.visible').and('be.empty');
+      });
+  });
+
+  it('displays `owners: Vec<AccountId>` input correctly', () => {
```

Comment:
we already have tests for this functionality in the mother spec.
what would be interesting for the multisig is to test some messages that haven't been tested before. maybe we should have a more detailed talk about this

Reply:
Just fixed the PRs CI issue. Should work now. Let me know what should be tested with the for the multisig contract. Maybe this PR does not include much of value for the repo. 🤔 

As you've mentioned, it might be best to have a single ink contract offering all possible in/output structures. Briefly worked on such a contract already.